addons/delivery/models/product_template.py

```diff
@@ -11,3 +11,8 @@ class ProductTemplate(models.Model):
         string="HS Code",
         help="Standardized code for international shipping and goods declaration. At the moment, only used for the FedEx shipping provider.",
     )
+    country_of_origin = fields.Many2one(
+        comodel_name='res.country',
```

Comment:
no need to define the kwargs here since it's always the same in a M2O case. Keep the standard